CHANGELOG.md

```diff
@@ -3,22 +3,36 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
 ## Unreleased
 
+## [0.6.0-pre.2] - 2024-10-15
+
 ### Changed
 
-- Bumped `crypto-bigint` to 0.6.0-rc.6 (pinned) and MSRV to 1.81. ([#55])
+- Bumped `crypto-bigint` to 0.6.0-rc.6 and MSRV to 1.81. ([#55])
```

Comment:
I'm not sure it's worth enumerating internal changes in the changelog. Definitely not in the `Changed` section, it's intended for API-breaking changes. #54 probably qualifies for `Added` (improved performance), the rest I think can be safely omitted.

Reply:
Maybe even `Fixed`, not `Added`

Reply:
MSRV changes are considered breaking changes so that one needs to stay.

I think it's worthwhile pausing here for a second and consider what process we want here. As always it's a trade-off: we can spend time pondering each PR and discuss&decide if it's "CHANGELOG-worthy", or we can spend less time on it and simply include all merged PRs since the last release.

I went with the latter approach here, which I think is sensible for a pre-1.0 crate, but if you prefer spending the time required to make the call on what is worth mentioning and what isn't, then that's fine too.

Feel free to push to this branch so I can look&learn about what your preferences are.

Reply:
I agree about MSRV, that should stay, I was talking about the new entries.

I don't think a lot of pondering is required here. As someone who reads changelogs, I am not interested in the internal changes, I only want to know what affects me as a user. Things like the change in `rustfmt` settings only add noise and don't affect how I use the library. If think we absolutely must include every PR in the changelog, let's at least make an `Internal` section for those kind of changes.

Reply:
I think not having clear rules as to what goes into the CHANGELOG is going to slow us down and be annoying, so let's establish a criterion: only include PRs that affect the public API and/or general user experience (e.g. performance).